addons/account/models/account_move.py

```diff
@@ -2030,6 +2030,8 @@ def compute_tax_repartition_lines_mapping(move_vals):
             elif line_vals.get('tax_repartition_line_id'):
                 # Tax line.
                 invoice_repartition_line = self.env['account.tax.repartition.line'].browse(line_vals['tax_repartition_line_id'])
+                if invoice_repartition_line not in tax_repartition_lines_mapping:
+                    raise UserError(_("It seems that the taxes have been modified since the creation of the journal entry. You should create the credit note manually instead."))
```

Comment:
don't forget to update the translation file

Reply:
I've never done it before. How does it work ?

Reply:
It is done.